samples/debugmalloc/enclave/enc.c

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/stdlib.h>
+#include <openenclave/debugmalloc.h>
+#include <stdio.h>
+
+// Include the trusted debugmalloc header that is generated
+// during the build. This file is generated by calling the
+// sdk tool oeedger8r against the debugmalloc.edl file.
+#include "debugmalloc_t.h"
+
+// This is the function that the host calls. It prints
+// a message in the enclave before calling back out to
+// the host to print a message from there too.
+void enclave_hello()
+{
+    void *buf1, *buf2, *buf3;
+    uint64_t num;
```

Comment:
OE style, especially in samples, is to avoid abbreviations.
```suggestion
    void *buffer1, *buffer2, *buffer3;
    uint64_t count;
```